### PR TITLE
UHM-5066, migrate HIV_EXAM_SYMPTOMS

### PR DESCRIPTION
**Pull request by cioan**
I still need to add a few more rows in the hivmigration_symptoms_map to complete the mapping of the symtoms names, but just wanted to make sure the model of this migration looks right to you. Thanks!

### CODE REVIEW

File: etl/src/main/java/org/pih/hivmigration/etl/sql/ExamSymptomsMigrator.groovy [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+            from hivmigration_exam_symptoms s, hivmigration_symptoms_map m  
+            where s.symptom is not null and s.symptom = m.hiv_symptom 
+                and m.concept_source_map != '' and m.openmrs_concept_code != ''; 
+
+            -- Create Non-Coded Symptom Name           
+            INSERT INTO tmp_obs (
+                obs_group_id,                
+                source_encounter_id, 
+                concept_uuid,
+                value_coded_uuid,
```
Comment (brandones) [inline etl/src/main/java/org/pih/hivmigration/etl/sql/ExamSymptomsMigrator.groovy:147]: I think this is is supposed to be `value_text`

Comment (brandones) [inline etl/src/main/java/org/pih/hivmigration/etl/sql/ExamSymptomsMigrator.groovy:147]: I think this is is supposed to be `value_text`

**Comment by brandones** [comment]
Looks good to me so far! Ping when it's all the way done?

**Review by mogoodrich — APPROVED** [review]
@cioan  LGTM, though admittedly my review hasn't been super thorough since with this migrations I've put the majority of my time into testing.

**Comment by cioan** [comment]
Thanks @brandones and @mogoodrich ! This PR is complete now and it just finished running on my machine too. 
@mogoodrich : I appreciate the very thorough testing. Thanks!

**Pull request merged**